FlowCrypt/Controllers/SideMenu/Main/SideMenuNavigationController.swift

```diff
@@ -57,6 +63,7 @@ final class SideMenuNavigationController: ENSideMenuNavigationController {
             $0.delegate = self
             $0.animationDuration = Constants.animationDuration
         }
+        refreshKeysFromEKMIfNeeded(context: appContext)
```

Comment:
Will this run only once per app run? I imagine this could run each time the view is rendered which could be more than once per app startup?

Reply:
Not once per app run.
Each time app runs + when user logs into new account.

Reply:
I guess also when user switches accounts? (in left menu?)

Reply:
Yeah, correct